tests/unit/tasks/install-blueprint-test.js

```diff
@@ -33,28 +33,28 @@ describe('InstallBlueprintTask', function() {
       td.when(task._createTempFolder()).thenResolve(task._tempPath);
     });
 
-    it('resolves "foobar" by looking up the "foobar" blueprint locally', function() {
+    it('resolves "foobar" by looking up the "foobar" blueprint locally', async function() {
       let foobarBlueprint = { name: 'foobar blueprint' };
       td.when(task._lookupBlueprint('foobar')).thenResolve(foobarBlueprint);
 
-      return expect(task._resolveBlueprint('foobar')).to.eventually.equal(foobarBlueprint);
+      expect(await task._resolveBlueprint('foobar')).to.equal(foobarBlueprint);
     });
 
     it('rejects invalid npm package name "foo:bar"', function() {
       let error = new Error('foobar not found');
       td.when(task._lookupBlueprint('foo:bar')).thenReject(error);
 
-      return expect(task._resolveBlueprint('foo:bar')).to.be.rejectedWith(error);
+      expect(task._resolveBlueprint('foo:bar')).to.be.rejectedWith(error);
```

Comment:
`rejectedWith` calls need an `await`

Reply:
Yeah, wanted to get you opinion about how to handle test cases with `.to.rejectedWith` assertions first. Will update the file shortly.